api/api.md

```diff
@@ -9,6 +9,8 @@ The base URL will be https://`siteaddress`/external/api/rest/v2/
 
 All endpoints in the document are relative to this url.
 
+Note: The term "rooms" has been amended to "spaces" any references to a room or rooms will now be referred to as space or spaces. This change does not affect the current endpoints.  
+
```

Comment:
Note: The term "room" has been amended to "space"; all references to a "room" or "rooms" will now be referred to as a "space" or "spaces". This change does not affect current endpoints.

Reply:
@Jben13 after you make this change, don't forget to run `make export` and commit the new index.html file too

Reply:
@RedStarThrower @rmoscowitz 
Thanks for the input. I have update the changes to the note.